### PR TITLE
HACK: Redefine free() to fix bug in AXRuntime.framework

### PR DESCRIPTION
**Pull request by mmmulani**

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -28,11 +31,13 @@ end
 # XCT Testing
 
 target :XCTWebDriverAgentLib, :exclusive => true do
+  pod 'fishhook', :git => 'https://github.com/facebook/fishhook.git', :commit => '8dbd09b'
```
Comment (marekcirkos) [inline Podfile]: Do we needed in XCT as well?

Comment (mmmulani) [inline Podfile]: I'm not sure but I think so because we share the same main.m? was more afraid of breaking it to be honest :P

**Comment by marekcirkos** [comment]
This insanity looks good to me :)
image

**Comment by Jonahss** [comment]
Mad props

**Pull request merged**